### PR TITLE
Silently validate the geographic fields at form initialization

### LINKED ISSUE #23: [DataDetail] The map fields must be updated to enable the submit button
When we have a creation form with a facultative geographic field, the submit button doesn't be enabled when we update one other fields, contrary to what normally happens.

So we need to validate the geographic fields at initialization to check if the value is valid or not, as it is already implicitly done for components that use the v-model directive.

### PR DESCRIPTION
**Pull request by jcougnaud**
Closes #23

### CODE REVIEW

File: front/src/components/ui/ColumnType/File/Input.vue [hunk 1]
```diff
@@ -110,7 +110,6 @@ export default {
   },
   watch: {
     attachments: {
-      immediate: true,
```
Comment (DarkVisenya) [inline front/src/components/ui/ColumnType/File/Input.vue:113]: Why did you remove this ?

Comment (jcougnaud) [reply front/src/components/ui/ColumnType/File/Input.vue:113]: If I remember correctly, the problem here was that if the field was required and empty, an error message was displayed when the form opened, asking the user to enter a value.

So I changed the way it worked to silently validate the field when it was loaded (@hook:mounted), in the same way as the geographic fields.
